core/src/test/java/org/elasticsearch/search/builder/SearchSourceBuilderTests.java

```diff
@@ -214,6 +215,15 @@ public static void afterClass() throws Exception {
         namedWriteableRegistry = null;
     }
 
+    private static NamedWriteableRegistry newNamedWriteableRegistry() {
+        NamedWriteableRegistry namedWriteableRegistry = new NamedWriteableRegistry();
+        // the TermSuggestionBuilder, PhraseSuggestionBuilder, etc. are registered via the SearchModule
+        namedWriteableRegistry.registerPrototype(SmoothingModel.class, Laplace.PROTOTYPE);
+        namedWriteableRegistry.registerPrototype(SmoothingModel.class, LinearInterpolation.PROTOTYPE);
+        namedWriteableRegistry.registerPrototype(SmoothingModel.class, StupidBackoff.PROTOTYPE);
```

Comment:
I think registering the smoothing models should also go somewhere into SearchModule. Since they are only used in PhraseSuggestionBuilder, I would probably put them alongside the build-in suggestion builders in `configureSuggesters()`